lib/ansible/modules/blockinfile.py

```diff
@@ -90,6 +90,14 @@
     type: str
     default: END
     version_added: '2.5'
+  wrap_with_blank_lines:
```

Comment:
How about `newline_padding`?

Reply:
It was suggested by @mkrizek that this might be better as two options: `prepend_newline` and `append_newline`. I like that better than `wrap_with_blank_lines` or my suggestion above.

Reply:
Yeah absolutely! I will break it in two different options and adapt the tests to check both cases.